container_pipeline/workers/base.py

```diff
@@ -29,16 +29,17 @@ def notify(self, data):
         """
         self.queue.put(json.dumps(data), 'master_tube')
 
-    def export_logs(self, logs, destination):
+    def export_logs(self, logs, destination, json_logs=False):
```

Comment:
On a second thought, why do we need ``json_logs`` here and why should this method care about jsonifying things. Just send some text, and it will write to a file. And, it's not like that if you set ``json_logs`` to ``True`` for random text log ``logs``, it will work. I was thinking that we'll be able to export logs in text as well as ``json`` format with this change. This doesn't seem to be the case here.